modern/src/maki-interpreter/objects.js

```diff
@@ -2171,7 +2171,7 @@ const objects = {
       {
         parameters: [],
         name: "isRunning",
-        result: "",
+        result: "boolean",
```

Comment:
Can you add a comment here explaining that this value deviates from the original derived version of this?

Reply:
I added a note to the "Quirks" section here: https://paper.dropbox.com/doc/Maki-Language-Spec--AkgMmb9ZE1U_Cl6CNX4IFCyHAg-csainvAwSr3SBUXO5DWXy

Reply:
Yup, will definitely add that comment explaining. Thanks for updating the paper doc!